contracts/utils/SignatureValidator.sol

```diff
@@ -26,6 +26,7 @@ contract SignatureValidator {
   // Allowed signature types.
   uint256 private constant SIG_TYPE_EIP712 = 1;
   uint256 private constant SIG_TYPE_ETH_SIGN = 2;
+  uint256 private constant SIG_TYPE_WALLET_BYTES = 3;
```

Comment:
Maybe rename to `SIG_TYPE_WALLET_BYTES32` instead since this is for the hash version of isValidSignature